wordpress-mobile/WordPress-Android
# [Jetpack Social] Remove feature flag

Fixes #19576 

To test:

Taken from https://github.com/wordpress-mobile/WordPress-Android/pull/18897

1 - Open the app
2 - Go to Posts
3 - Open a Draft post / create a new post
4 - Open the Post Settings
5 - Verify the JP Social section is available
6 - Hit Publish
7 - Verify the JP Social item is available in the pre-publish sheet

## Regression Notes
1. Potential unintended areas of impact
Jetpack social section on edit post flow

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and updated unit tests

3. What automated tests I added (or what prevented me from doing so)
Updated `EditorJetpackSocialViewModelTest.kt`

PR submission checklist:


UI Changes testing checklist:
